### ISSUE TITLE
QUaServer currently does not support importing nodes directly from standard OPC UA XML files.

### ISSUE THREAD

**Issue by zi-yue-01** [juangburgos/QUaServer#63]

**Comment by zi-yue-01** [comment]
Hello, I noticed that the file 09_Serialization config.xml is not a standard OPC UA XML file. May I ask if QUaServer supports importing nodes from standard OPC UA XML files?

**Comment by juangburgos** [comment]
No, it does not implements standard OPC UA XML files, but it provides the means for you to implement it by defining the required `writeInstance` and `readInstance` interfaces, see [the docs](https://github.com/QUaServer/QUaServer?tab=readme-ov-file#Serialization) and serialization examples.

If you create an implementation for standard OPC UA XML files we would gladly merge it into the repo.
